node/src/components/contract_runtime/operations.rs

```diff
@@ -191,6 +242,90 @@ pub fn execute_finalized_block(
                         .cloned()
                 },
             );
+
+    // Pruning
+    //
+    // Currently disabled as `key_block_height_for_activation_point` might not be available on a
+    // freshly-joined node at the time this call to execute block happens.
+    if false {
+        if let Some(previous_block_height) = finalized_block.height().checked_sub(1) {
+            // The following three values should be passed into this function if pruning is
+            // re-enabled. For now, they're set so that `calculate_prune_eras` will panic.
+            let activation_point_era_id = EraId::default();
+            let key_block_height_for_activation_point = u64::MAX;
+            let prune_batch_size = 1;
```

Comment:
Do we want to remove this from chainspec? I thought you could set it to zero when we don't need to prune anything in a given protocol version (see also changelog comment)